extension/src/popup/components/ModalInfo/styles.scss

```diff
@@ -24,10 +34,10 @@
 
   &--subject {
     font-size: 0.75rem;
-    color: var(--color-gray-70);
-    background-color: var(--color-gray-30);
+    color: #a0a0a0;
```

Comment:
Should we add these colors to the SDS? These are the ones used in the mocks.

Reply:
Yeah, I think so - but it's probably okay to leave as-is for now. We need to update quite a few SDS components/colors. I'm working with Joseph to try to schedule this for Q3